modules/ablidaBidAdapter.js

```diff
@@ -0,0 +1,93 @@
+import * as utils from '../src/utils';
+import {config} from '../src/config';
+import {registerBidder} from '../src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'ablida';
+const ENDPOINT_URL = 'https://bidder.ablida.net/prebid';
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.placementId);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @return Array Info describing the request to the server.
+   * @param validBidRequests
+   * @param bidderRequest
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    if (validBidRequests.length === 0) {
+      return [];
+    }
+    return validBidRequests.map(bidRequest => {
+      const sizes = utils.parseSizesInput(bidRequest.sizes)[0];
+      const size = sizes.split('x');
+      const jaySupported = 'atob' in window && 'currentScript' in document;
+      const device = getDevice();
+      const payload = {
+        placementId: bidRequest.params.placementId,
+        width: size[0],
+        height: size[1],
+        bidId: bidRequest.bidId,
+        category: bidRequest.params.category,
```

Comment:
Is this one of the potential params that could be specified in the bidder params?  If so, can you please document this in your md file (and docs PR if needed)?